CHANGES.md

```diff
@@ -1,3 +1,19 @@
+## edge-20.2.1
+
+This edge release is a release candidate for `stable-2.7` and fixes an issue
+where the proxy could consume inappropriate amounts of memory.
+
+* Proxy
+  * Fixed a bug in the proxy's logging subsystem that could cause the proxy to
+    consume memory until the process is OOMKilled, especially when the proxy was
+    configured to log diagnostic information
+  * Fixed properly emitting `grpc-status` headers when signaling proxy errors to
+    gRPC clients
+* Internal
+  * Updated to Rust 1.40
```

Comment:
I don't think this is a user-facing change, so I probably wouldn't put it in the changelog?

Reply:
the gRPC behavior? it's a behavior change so I think it's "user" facing